strawberryfields/compilers/fock.py

```diff
@@ -64,6 +64,7 @@ class Fock(Compiler):
         "CXgate": {},
         "CZgate": {},
         "MZgate": {},
+        "sMZgate": {},
```

Comment:
would it be too much to ask to a Symplectic description of the `sMZgate` into the Gaussian compiler?